skyportal/tests/frontend/test_sources.py

```diff
@@ -299,32 +299,27 @@ def test_super_user_can_delete_unowned_comment(
     driver.wait_for_xpath(f'//div[text()="{public_source.id}"]')
     comment_box = driver.wait_for_xpath("//input[@name='text']")
     comment_text = str(uuid.uuid4())
+    driver.scroll_to_element_and_click(comment_box)
     comment_box.send_keys(comment_text)
     driver.scroll_to_element_and_click(
-        driver.find_element_by_xpath('//*[@name="submitCommentButton"]')
+        driver.wait_for_xpath('//*[@name="submitCommentButton"]')
     )
-    try:
-        comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
-    except TimeoutException:
-        driver.refresh()
-        comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
+
     driver.get(f"/become_user/{super_admin_user.id}")
     driver.get(f"/source/{public_source.id}")
-    driver.refresh()
+
     comment_text_div = driver.wait_for_xpath(f'//div[./p[text()="{comment_text}"]]')
     comment_div = comment_text_div.find_element_by_xpath("..")
     comment_id = comment_div.get_attribute("name").split("commentDiv")[-1]
-    delete_button = comment_div.find_element_by_xpath(
+
+    # wait for delete button to become interactible - hence pause 0.1
+    ActionChains(driver).move_to_element(comment_text_div).pause(0.1).perform()
+
+    delete_button = driver.wait_for_xpath(
         f"//*[@name='deleteCommentButton{comment_id}']"
     )
-    driver.execute_script("arguments[0].scrollIntoView();", comment_div)
-    ActionChains(driver).move_to_element(comment_div).perform()
-    driver.execute_script("arguments[0].click();", delete_button)
-    try:
-        driver.wait_for_xpath_to_disappear(f'//p[text()="{comment_text}"]')
-    except TimeoutException:
-        driver.refresh()
-        driver.wait_for_xpath_to_disappear(f'//p[text()="{comment_text}"]')
+    driver.scroll_to_element_and_click(delete_button)
+    driver.wait_for_xpath_to_disappear(f'//p[text()="{comment_text}"]')
```

Comment:
For some reason, I've had issues with disappear.  Looks like it works now! :man_shrugging: